custom/ewsghana/api.py

```diff
@@ -689,6 +750,15 @@ def sms_user_sync(self, ews_smsuser, **kwargs):
         if ews_smsuser.role:
             sms_user.user_data['role'] = [ews_smsuser.role]
 
+        if ews_smsuser.phone_numbers:
+            sms_user.user_data['connections'] = []
+            for connection in ews_smsuser.phone_numbers:
+                sms_user.user_data['connections'].append({
```

Comment:
ok, sounds good. we can probably purge that connection info from each user once the migration is all over